recipes-core/images/xenclient-installer-image.bb

```diff
@@ -38,7 +39,6 @@ IMAGE_INSTALL = "\
     packagegroup-base \
     packagegroup-xenclient-common \
     packagegroup-xenclient-installer \
-    kernel-module-e1000e \
```

Comment:
Is this now built in?

Reply:
The kernel used for the installer has all kernel modules built in. OTOH if this is an out-of-tree kernel module that is not part of the standard linux distribution, then it is not built because OE can't handle the 32/64-bit discrepancy between the build environment and the 64-bit kernel.

Reply:
Ah, okay.  You did INSANE_SKIP_kernel-module-txt = "arch" elsewhere, and I recognized it for the mixed 32/64 compilation.

Reply:
Correct, I believe we tried that for the kernel modules too and it would still fail so that's why ended up compiling everything into the kernel.

Reply:
@tklengyel , I just pushed a branch here: https://github.com/jandryuk/xenclient-oe/commits/stable-6-64bit with changes for building 64bit compat kernels and modules.
'ERROR_QA_remove += "arch" ' lets bitbake build and package 64bit modules from the kernel recipe.  You could do the same if you want to use modules with the installer kernel.

Reply:
@jandryuk thanks! At the moment I don't believe there is a problem with the selection of kernel modules for the installer, so unless someone can point to a hardware that needs a special out-of-tree kernel module for the installer to work, I think it's safe to keep things as is.